torch/_C/_distributed_c10d.pyi

```diff
@@ -122,7 +122,7 @@ class Work:
     def is_completed(self) -> bool: ...
     def is_success(self) -> bool: ...
     def exception(self) -> Any: ...
-    def wait(self, timeout: timedelta) -> bool: ...
+    def wait(self, timeout: timedelta = ...) -> bool: ...
```

Comment:
Didn't know ellipsis could be used for default values, but wouldn't it be better to give it an actual default?

Reply:
> Didn't know ellipsis could be used for default values, but wouldn't it be better to give it an actual default?

The default value (kNoTimeout in distributed/c10d/init.cpp) is not exported to Python, should I export this value and add it as the default?

Reply:
make senes to add it since default no timeout is actually affecting lots of internal behavior of processgroup